queries/translations.sql

```diff
@@ -29,8 +30,9 @@ ON CONFLICT (episodes_id, languages_code) DO UPDATE SET title       = EXCLUDED.t
 -- name: ListSeasonTranslations :many
 WITH seasons AS (SELECT s.id
                  FROM seasons s
-                          LEFT JOIN shows sh ON sh.id = s.show_id
-                 WHERE s.status = ANY ('{published,unlisted}')
+                          JOIN shows sh ON sh.id = s.show_id
+                 WHERE s.translations_required
```

Comment:
if I set a field named `translations required´ to false I would expect it to still use existing translations.
Opposite of required is "optional".

Reply:
Translations aren't retrieved from this query. This is only for the sync.